.eslintrc.yml

```diff
@@ -176,6 +176,7 @@ rules:
   react/prefer-stateless-function: off
   react/prop-types: error
   react/react-in-jsx-scope: error
+  react/require-optimization: warn
```

Comment:
This rule will throw up a warning for any components which don’t have a defined `shouldComponentUpdate` method. Realistically, most of our components should be using `shallowCompare` eventually so that they don’t re-render unless things change!